integreat_cms/cms/models/abstract_content_model.py

```diff
@@ -367,7 +394,7 @@ def get_translation_state(self, language_slug):
         :return: A string describing the state of the translation, one of :data:`~integreat_cms.cms.constants.translation_status.CHOICES`
         :rtype: str
         """
-        translation = self.get_major_public_translation(language_slug)
+        translation = self.get_major_translation(language_slug)
         if not translation:
             if self.fallback_translations_enabled:
                 fallback_translation = self.get_public_translation(
```

Comment:
```suggestion
                fallback_translation = self.get_translation(
```
I guess to be consistent here, we'd need to show the "fallback" icon always if there is a fallback translation, independently from their publication status?
This sure might be a bit confusing in the beginning, because if the fallback is a draft, then also the non-existing translation is not shown in the app. But if we're being strict here, we should also make the fallback status independent from publication...